scripts/get-prod-db.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -euxo
```

Comment:
```suggestion
set -euxo pipefail
```

https://wizardzines.com/comics/bash-errors/

Reply:
This is a trade off...I'm using `sh` not `bash` in this script, basically to keep the feature set small enough as to not run in to cross-OS compatibility issues. I also think `sh` reduces the risk of errors / security problems that `bash` introduces.

In general, I think `sh` is a good set of features that we should stick to, rather than `bash`. 

I'm happy to break this if we need `bash`, but in this case I think `sh` is better.

Because we're also not expecting to pipe this command to anything else, I don't think we need to worry about failures in a pipe.

Reply:
This might be a good one to make a decision on if the direction we're going in is writing a lot more shell scripts. Lets park this specific thread for now anyway.